modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
Do we use `formatFileSize.js` anywhere? If not we should remove it but if we do `components/FileSize.js` should use it instead of `filesize` directly.